### PR TITLE
MM-41988 Updated links to legacy domain about.mm.com

### PR DESCRIPTION
**Pull request by cwarnermm**
#### Summary
Updated in-product links to legacy domain about.mattermost.com.

#### Release Note

```release-note
Updated in-product links to legacy domain about.mattermost.com
```

### CODE REVIEW

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Carrie! Could you please check other similar links as a result when searching for `about.mattermost.com`?

**Comment by cwarnermm** [comment]
@saturninoabril Thanks! I believe I've updated the remaining legacy URLs.

**Review by isacikgoz — APPROVED** [review]
LGTM 👍

File: i18n/bg.json
Comment (saturninoabril) [inline i18n/bg.json:2476]: (non-blocking for me) I believe we are only changing translation strings at `en.json` and Translation team will follow through the necessary changes per language owner. So, I don't know if this could be an exception.

**Review by saturninoabril — APPROVED** [review]
Thanks Carrie! Left one comment but non-blocking for me. Feel free to merge.

File: model/config.go
Comment (hmhealey) [inline model/config.go:131]: @cwarnermm Like I mentioned in the web app PR, we need to keep these old redirects around since almost all existing Mattermost instances will be pointing to them.

I still think it's a good idea to have them pointing to redirects as well for a similar reason, but that's not as necessary

**Review by hmhealey — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md
Comment (cwarnermm) [inline .github/ISSUE_TEMPLATE.md:3]:
```suggestion
For troubleshooting see: https://forum.mattermost.com/.
```

File: .github/ISSUE_TEMPLATE.md
Comment (cwarnermm) [inline .github/ISSUE_TEMPLATE.md:4]:
```suggestion
For feature proposals see: https://www.mattermost.com/feature-ideas/
```

File: .github/ISSUE_TEMPLATE.md
Comment (cwarnermm) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
For feature proposals see: https://www.mattermost.com/feature-ideas/
```

**Comment by cwarnermm** [comment]
@hmhealey & @saturninoabril - I believe that all paused/outstanding work on this PR has now been completed. I have removed all of the language files I touched with URL updates (except the en.json file), and I replaced the about.mm.com legacy URLs you flagged as potentially problematic with the new destination links Emily made available. Removing the "Do not merge" label for final reviews.

**Comment by hmhealey** [comment]
Sorry, by removing the other language files, I meant removing the changes to them and reverting to the existing versions which is why these are all conflicting now. Let me know if you need help doing that since it looks like you're using the web interface to modify files, and I'm not sure if there's an easy way to do that from here

**Review by hmhealey — APPROVED** [review]

**Comment by cwarnermm** [comment]
@hmhealey - I identified and fixed a couple of strings. I believe we're good to go now.

**Comment by saturninoabril** [comment]
@cwarnermm Build failed due to failed tests. I'll defer to @isacikgoz if you need help in sorting out those tests.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Looks good to me, just need to address failed tests. Thanks Carrie for updating!

**Comment by cwarnermm** [comment]
@isacikgoz - Would you be able to assist with the build failures, please?

**Review by saturninoabril — APPROVED** [review]
Thanks @cwarnermm and @isacikgoz! LGTM.

File: i18n/en.json
Comment (saturninoabril) [inline i18n/en.json:1019]: @cwarnermm @emdecr I noticed while reviewing webapp changes that this redirects to `https://docs.mattermost.com/guides/channels.html#work-with-channels`. Should that be used here instead?

**Comment by cwarnermm** [comment]
@saturninoabril - Thanks for calling this out! Yes, this URL should likely be updated. The good news is that we have a redirect in place to catch this in the meantime and take the user to the correct page in the docs. Would you possibly be open to updating that link? Otherwise, please log this issue and assign it to me.

**Comment by saturninoabril** [comment]
Hello @cwarnermm, you're welcome and sure, I'll take care of it since it will require a follow up change to webapp E2E as well. And no worries 😉.

Great job by the way!

**Pull request merged**
